llvm/test/CodeGen/AMDGPU/coalescer_remat.ll

```diff
@@ -12,7 +12,7 @@ declare float @llvm.fma.f32(float, float, float)
 ; CHECK:  v_mov_b32_e32 v{{[0-9]+}}, 0
 ; CHECK:  v_mov_b32_e32 v{{[0-9]+}}, 0
 ; It's probably OK if this is slightly higher:
-; CHECK: ; NumVgprs: 8
+; CHECK: ; NumVgprs: 5
```

Comment:
I could use another pair of eyes on this test. From what I can tell, it uses v[4:7] (`buffer_store_dwordx4 v[4:7], off, s[0:3], 0`) but only defines v4. Am I missing smth, or is this just broken?

Reply:
The first store in the loop is doing a vector store with only the 0th element defined